examples/tests/ExceptIn/edits

```diff
@@ -0,0 +1,36 @@
+# This edit works.
+except in ExceptIn.bar, ExceptIn.baz rename value ExceptIn.A = ExceptIn.X
+
+
+# The following edit has a couple of issues; see below for more information.
+except in ExceptIn.Test1, ExceptIn.bar, ExceptIn.baz rename type ExceptIn.Test1 = ExceptIn.Test2
+
+# 1.
+# In the type signature of ExceptIn.bar and ExceptIn.baz, Test1 is renamed to
+# Test2, even though we excluded ExceptIn.bar and ExceptIn.baz when we specified
+# the rename edit.
+
+# This is not an issue specific to "except in" edits, but rather with the fact that
+# local edits are not taken into account when translating signatures. This has been
+# documented; see issue #156 on github (https://github.com/antalsz/hs-to-coq/issues/156).
```

Comment:
> I am curious though how this edit would interact with other edits such as `rename module`?

In this case, I think `hs-to-coq` will simply perform the `rename module` edit globally as if `except in` hadn't been used. This is actually the behavior for most kinds of edits, since most edits seem to apply "globally" and thus it doesn't make sense to exclude them from a particular definition. Currently, only a few edits are actually affected by `except in`; those are `skip case patterns`, `rename`, and `rewrite`. However, my understanding of most edits is limited and there could be other edits to which `except in` could apply; if this is the case, let me know!

Reply:
If a module has been renamed, should the original name or the renamed name be used in `except in`? I suppose the answer is the same as `in` (and if I understand correctly, the renamed name should be used?), but we have bugs like #137 so it's worth checking :)

Reply:
Based on some testing, it seems that the **original** name has to be used in `except in`. This seems counterintuitive, and I think it's the same as what's happening in #137. In other words, I don't think it is specific to `except in` edits.